### PR TITLE
redirect runnable-jar-files 20.0.0.7/8

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: doc-redirects.properties [hunk 1]
```diff
@@ -27,6 +27,11 @@
 /docs/ref/command/*=/docs/latest/reference/command/
 /docs/ref/feature/*=/docs/latest/reference/feature/
 
+# Redirects for a file name change to runnable-jar-files.html. 20.0.0.8 was the latest version with the old file name.
+/docs/runnablejarfiles.html=/docs/runnable-jar-files.html.html
```
Comment (mbroz2) [inline doc-redirects.properties]: this would be
`/docs/ref/general/runnablejarfiles.html=/docs/runnable-jar-files.html.html`

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: doc-redirects.properties (hunk 1 above)
Comment (dmuelle) [reply doc-redirects.properties]: The runnable jar files topic is in the ROOT module, not reference like the feature and server overviews. Did it still need `/ref` in the URL? I can't quite remember how the URL structure has evolved so just want to make sure.

Comment (mbroz2) [reply doc-redirects.properties]: prior to antora it was at `/docs/ref/general/runnablejarfiles.html` 
it was never at `/docs/runnablejarfiles.html`

I don't expect this one to matter much as I can't imagine anyone having a bookmark to that after 1+ years and to my knowledge nothing references that old url.  However, doesn't hurt to have.

Comment (mbroz2) [reply doc-redirects.properties]: https://web.archive.org/web/20191013144912/https://openliberty.io/docs/ref/general/

Comment (dmuelle) [reply doc-redirects.properties]: thanks- updated

**Review by mbroz2 — APPROVED** [review]

**Comment by mbroz2** [comment]
For staging and prod, consider using the existing redirect branches to make the change in
https://github.com/OpenLiberty/docs-playbook/tree/redirects-staging
https://github.com/OpenLiberty/docs-playbook/tree/redirects-prod

**Pull request merged**